### ISSUE TITLE
Rename "Appointments" to "Appointment slots"

### ISSUE THREAD

**Issue by ChristophWurst** [nextcloud/calendar#3736]
**Is your feature request related to a problem? Please describe.**

We recently introduced a new feature that we call appointments. The term is hard to distinguish from events/meetings, translated into other languages it becomes even harder to find a distinguishable and descriptive term.

**Describe the solution you'd like**

Rename *Appointments* to *Appointment slots* or something else that is more descriptive and easier to translate.

**Describe alternatives you've considered**

N/a

**Additional context**

@rakekniven brought this up in the chat. 

cc @Valdnet @simonspa @jancborchardt @nimishavijay

**Comment by ChristophWurst** [comment]
In terms of timing I would say v3 should be released/translated as is. The rename can be done for v3.1 at the earliest.

**Comment by Valdnet** [comment]
When translated into Polish, the word "Appointments" is translated as "Spotkanie" ("Meeting"), which is the correct word in the English language.

On the other hand, the words "Appointment slots" can be translated into Polish, because "Slots" can be translated as: "Port" ("Port"), "Gniazdo" ("Socket"), "Otwór" ("Hole") and "Przedział czasowy" ("Time interval") which fits the most.

I translate the words "Appointment slots" as "Termin spotkań" ("Meeting dates") and I have the greatest difficulty with it in individual text strings.

To sum up:
"Appointments" is the correct word for "Meeting".
On the other hand, the word "Appointment slots" is less suitable for the Polish language.

**Comment by ChristophWurst** [comment]
Thanks for the insights.

Just to be sure we are talking about the same feature. It's this: https://docs.nextcloud.com/server/latest/user_manual/en/groupware/calendar.html#appointments. This will be released just tomorrow. You don't see this in the UI if you use Nextcloud Calendar v2.4

**Comment by simonspa** [comment]
Just some ideas:

For providing user:

* **Create new event** (ex. German: "ein Ereignis/eine Veranstaltung erstellen/anlegen"
* **Create free/bookable appointments**? (ex. German: "Freie/buchbare/reservierbare Termine anlegen/konfigurieren")
* **Available/bookable appointments**? (ex. German: "Verfügbare/buchbare Termine")
* **Appointment slots** (ex. German: "Terminfenster/Terminangebote"

For applying user:

* **book/arrange/make an appointment** (ex. German: "einen Termin buchen/vereinbaren")

**Comment by SebastianKrupinski** [comment]
Closing this as already implemented

<img width="321" height="231" alt="Image" src="https://github.com/user-attachments/assets/2310d8cb-2181-4133-bf69-5c49dfab4994" />
